vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -383,6 +385,9 @@ class HomeActivityViewModel @AssistedInject constructor(
             }
 
             val elementWellKnown = rawService.getElementWellknown(session.sessionParams)
+
+            mapUrlProvider.initWithWellknown(elementWellKnown)
```

Comment:
What about introducing a `WellknownService`, which should be used instead of the `RawService` to provide the contents of the wellknown file and get rid of all the wellknown related extensions for the `RawService`?  This should allow to cache the instance of the `ElementWellKnown` in memory which can be used without suspend functions?

Reply:
I provided an example implementation, used in `HomeActivityViewModel` and in the `UrlMapProvider`